File: kotlin/dataArrayTest.kt

fun test(){
    var a = arrayOf("apple", "banana")
    for(k in a){
        println(k)
    }
}

fun test2(){
	val a=intArrayOf(1, 2, 3)
	for(k in a){
		println(k)
	}
}

fun test3(){
	val a=IntArray(3, {i -> i + 2})
	for(k in a){
		println(k)
	}
}

File: go/dataArrayTest.go
package main


func main(){
	test()
}

func test(){
	var a [10]int
	a[0]=1
	a[1]=3
	
	for k, v := range a {
		println(k, v)
	}
}

func test2(){
	var a=[10]int{10, 20}

	for k, v := range a {
		println(k, v)
	}
}

func test3(){
	var a=[...]int{10, 20}
	
	for k, v := range a {
		println(k, v)
	}
}
